_docs/guides/create-notes-occurrences.md

```diff
@@ -0,0 +1,251 @@
+---
+title: How to create notes and occurrences
+overview: Describes how to create notes and occurrences once you have a running Grafeas server.
+
+order: 15
+
+layout: docs
+type: markdown
+---
+
+Before you can create [notes](/_docs/concepts/what-is-grafeas/overview.md) or
```

Comment:
I would recommend adding a "Before you begin" section:

## Before you begin

+ Set up a Grafeas server running on port 8080 using the instructions in [Running Grafeas](https://github.com/grafeas/grafeas/blob/master/docs/running_grafeas.md).